lib/utils.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Parse a MongodB connection URL, split it up and return an object
+ * containing the configuration properties.
+ * @param mongoConnectionURL The connection URL as String
+ * @returns Configuration Object
+ */
+exports.parseMongoConnectionURL = function (mongoConnectionURL) {
```

Comment:
not entirely sure about this function. This function is converting the connection string back to an object, only to allow the next function to convert the object back to connection string again.

Given that the database info is probably saved as object in fh-mbaas, perhaps `mongo_compat_api.js` can require the mongodb connection details to be passed in as an object?